hello-world/app/scripts/deploy_service.js

```diff
@@ -5,7 +5,7 @@ oasis.workspace.HelloWorld.deploy({
   header: {confidential: false},
 })
   .then(res => {
-    let addrHex = Buffer.from(res.address).toString('hex');
+    let addrHex = oasis.utils.bytes.toHex(res.address);
```

Comment:
Deployed HelloWorld at `0x0x930452709c8bbd55279a5e227acd761d7efbd147`

Reply:
exactly. it's all poorly specified. this method shouldn't be exposed :P

Reply:
I like the idea though. Since the printed address could be copied and used in other places, e.g., with a client, it makes sense to provide a more "official" way to generate it.